File: test/fixtures/langs/sample.java
public class Calc {
    //∷YAY⟨JAVA-1⟩
    //  unit: add
    //  intent: add two integers
    //  in: (a:number, b:number)
    //  out: number
    //  ensures: out === a + b
    //∷YAY-END⟨JAVA-1⟩
    public int add(int a, int b) {
        return a + b;
    }

    public int undocumented(int x) {
        if (x > 0) { return x; }
        return -x;
    }
}

File: test/fixtures/langs/sample.rb
#∷YAY⟨RB-1⟩
#  unit: add
#  intent: add two integers
#  in: (a:number, b:number)
#  out: number
#  ensures: out == a + b
#∷YAY-END⟨RB-1⟩
def add(a, b)
  a + b
end

def undocumented(x)
  if x > 0
    x
  else
    -x
  end
end
